test/azure/version-tolerant/Expected/AcceptanceTests/AzureBodyDurationVersionTolerant/bodydurationversiontolerant/aio/operations/_operations.py

```diff
@@ -30,7 +30,7 @@
 )
 
 T = TypeVar("T")
-JSONType = Any
+JSONObject = MutableMapping[str, Any]
```

Comment:
In `aio`, it seems that `JSONType` is defined but not used

Reply:
yep, thanks for catching! fixed